cg/apps/osticket.py

```diff
@@ -26,10 +26,10 @@ def __init__(self):
     def init_app(self, app: Flask):
         """Initialize the API in Flask"""
         self.setup(
-            api_key=app.config["OSTICKET_API_KEY"],
-            domain=app.config["OSTICKET_DOMAIN"],
-            osticket_email=app.config["SUPPORT_SYSTEM_EMAIL"],
-            email_uri=app.config["EMAIL_URI"],
+            api_key=app.config["osticket_api_key"],
```

Comment:
```suggestion
            api_key=app.config.osticket_api_key,
```
Does this not work as well?

Reply:
Unfortunately `app.config` is just a dict that I update with the contents of `app_config`. Hence we can only access with values via square brackets